apps/frontend/src/pages/common-layout/styles.module.css

```diff
@@ -0,0 +1,20 @@
+.container {
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	color: var(--color-text-primary);
+	background-color: var(--color-background-primary);
+}
+
+.container-sidebar-content {
+	display: flex;
+	flex-direction: row;
+	width: 100%;
+}
+
+.content-container {
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	padding: 0 80px 35px;
```

Comment:
As I remember on design all dimensions, padding and margins should be multiples of 8. This is a common pattern for layouts.